test/Microsoft.AspNetCore.SignalR.Microbenchmarks/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using BenchmarkDotNet.Running;
+
+namespace Microsoft.AspNetCore.SignalR.Microbenchmarks
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Contains("--debug"))
```

Comment:
I had to debug an issue in a benchmark, which I did by manually invoking it and used this to help launch the debugger. I usually just leave this in, but I can clean it up.